admin/javascript/src/components/accordion/styles.scss

```diff
@@ -0,0 +1,53 @@
+// Accordion styles
+.accordion-group {
+	margin-top: $spacer-y;
+
+	&__title {
+		margin-bottom: 0;
+
+		a {
+			/* Todo: extend table header */
+			font-size: $font-size-sm; 
+			line-height: $line-height-base; 
+			font-weight: 400; 
+			text-transform: uppercase; 
+			padding: $spacer-x*.75 $spacer-y;
+			/* end table header */
+			
+			display: block;
+			margin-left: -$spacer-y;
+			margin-right: -$spacer-y;
+			text-decoration: none;
+			position: relative;
+			border-bottom: 1px solid #ddd;
+			color: $body-color;
+
+			&::before {
+				padding: $spacer-x*.75 $spacer-y;
+				position: absolute;
+				right: 0;
+				top: 0;
+				font-size: $font-size-lg;
+				line-height: $line-height-base;
+				color: lighten($body-color,30);
+				text-align: center;
+				content: "7";
+				font-family: silverstripe;
+			}
+
+			&.collapsed::before {
+				content: "6";
```

Comment:
Not very readable way of referencing icons in the icon font - but looks like there's no straightforward way to fix this (found http://varya.me/en/posts/dealing-atomically-with-icon-fonts/). One significant disadvantage here: From what I can tell, you need access to a specific http://fontastic.me/ account (and icon collection) to add icons to that font. How would you see a community member contributing there?